### PR TITLE
Add Adventure Bar Story

### PR DESCRIPTION
**Pull request by Vocaless**
Adds Adventure Bar Story.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/adventure-bar-story/game.dat [hunk 1]
```diff
@@ -0,0 +1,20 @@
+title = "Adventure Bar Story"
+description = "Adventure Bar Story is and RPG published by Circle Entertainment, Rideon.\n\nYou play as Siela, a girl who inherits a bar from her dead parents. One day, a guy named Gustav comes to your bar and asks you to sell it to him, so that he can build his new restaurant in front of the palace. You refused the offer and now you are determined to make your bar more successful than his. You will fight monsters to collect ingredients, unlock new areas, meet new people and eat to level up."
```
Comment (Schplee) [inline games/adventure-bar-story/game.dat]: Please remove the line breaks from the description `\n`

File: games/adventure-bar-story/game.dat [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+game_type = "eshop"
+
+[[releases]]
+title = "0004000000160F00"
+region = "usa"
+release_date = "2015-02-19"
+
+[[testcases]]
+title = "0004000000160F00"
+compatibility = "0"
```
Comment (Schplee) [inline games/adventure-bar-story/game.dat]: Are you sure it's perfect? Graphics are 100% accurate, audio is 100% accurate? Nothing is broken/missing?

File: games/adventure-bar-story/game.dat (hunk 1 above)
Comment (j-selby) [inline games/adventure-bar-story/game.dat]: Don't need the newlines here.

File: games/adventure-bar-story/game.dat [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+date = "2017-08-12"
+version = "HEAD-343bbfd"
```
Comment (Schplee) [inline games/adventure-bar-story/game.dat]: This isn't a recent Nightly build. Please only test via the latest Nightly.

File: games/adventure-bar-story/game.dat [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+cpu = "Intel Core i7-6500U"
+gpu = "Intel HD Graphics 520"
+os = "Windows 10 14393.1593"
```
Comment (Schplee) [inline games/adventure-bar-story/game.dat]: `Windows 10 14393` is good enough.

**Comment by TheKoopaKingdom** [comment]
Closing this because of inactivity.

**Pull request closed without merging**